logging/src/main/java/org/wildfly/swarm/logging/LoggingFraction.java

```diff
@@ -51,6 +60,9 @@
 
     public static final String COLOR_PATTERN = "COLOR_PATTERN";
 
+    public static LoggingFraction loggingFraction;
```

Comment:
If you look to the code of the master branch, you will see that an instance of the LogginFraction is created within the static method "createDefaultLoggingFraction" - https://github.com/wildfly-swarm/wildfly-swarm/blob/master/logging/src/main/java/org/wildfly/swarm/logging/LoggingFraction.java#L121

As now we have a new static method where we will create a defaultFileLoggingFraction, then we should be able to reuse an existing instance of the LoggingFraction instead of creating a new object otherwise when we will chain the 2 static methods within (https://github.com/cmoulliard/wildfly-swarm-core/blob/83ae9cdb1bd3940805e07cf6b705e11a5cfdac8a/testsuite/testsuite-logging/src/test/java/org/wildfly/swarm/logging/FileLoggingInfoLevelArquillianTest.java#L58-L61), then the console will not be created. 

Do you have a better idea to code that ?